mockk/jvm/src/test/kotlin/io/mockk/jvm/HashMapMockTest.kt

```diff
@@ -0,0 +1,28 @@
+package io.mockk.jvm
+
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.assertDoesNotThrow
+import java.util.concurrent.ConcurrentHashMap
+import org.junit.jupiter.api.Test
+
+class HashMapMockTest {
+    @Test
+    fun concurrentHashMap_shouldBeSpied_Successfully() {
+        val map = spyk(ConcurrentHashMap<String, String>())
+        assertDoesNotThrow { map.put("key", "value")  }
+
+        verify(exactly = 1) { map.put("key", "value") }
+    }
+
+    @Test
+    @Disabled(value = "mocking of abstractMap don't work")
```

Comment:
Not sure if we want to keep this in the codebase disabled as a reference or remove it altogether, what do you think?

Reply:
I would keep it. Then we can look at the issue and enable the test

Reply:
Fine with me. Can you please just add a comment on why it's disabled?